tasks/db-init.ts

```diff
@@ -0,0 +1,88 @@
+import { createServer } from '../src/server'
+
+async function task() {
+    const [server, closeServer] = await createServer()
+
+    await server.db.query(createPlugin)
+    await server.db.query(createPluginAttachment)
+    await server.db.query(createPluginConfig)
+
+    await closeServer()
+}
+
+const ifNotExists = 'if not exists'
+
+const createPlugin = `
+    create table ${ifNotExists} posthog_plugin
+    (
+        id            serial  not null
+            constraint posthog_plugin_pkey
+                primary key,
+        name          varchar(200),
+        description   text,
+        url           varchar(800),
+        config_schema jsonb   not null,
+        tag           varchar(200),
+        archive       bytea,
+        from_json     boolean not null,
+        from_web      boolean not null,
+        error         jsonb
+    );
+`
+
+const createPluginAttachment = `
+    create table ${ifNotExists} posthog_pluginattachment
```

Comment:
Totally nitpicking, but conventionally SQL statements are ~~SCREAMING~~ uppercase, which is slightly clearer.

Reply:
SCREAMING implemented! I just uppercased all keywords and left the field types in place (`varchar`), even though they could scream as well...